src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -382,24 +397,146 @@ export const WorkspaceForm = ({
     setIcon(newIcon);
   }, []);
 
-  const handleDefaultVISThemeInputChange = useCallback<
-    Required<EuiComboBoxProps<string>>['onChange']
-  >((options) => {
-    setDefaultVISTheme(options[0]?.value);
+  const handleTabFeatureClick = useCallback(() => {
+    setTabFeatureSelected(true);
+  }, []);
+
+  const handleTabPermissionClick = useCallback(() => {
+    setTabFeatureSelected(false);
   }, []);
 
+  const onDefaultVISThemeChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setDefaultVISTheme(e.target.value);
+  };
+
+  const closeCancelModal = () => setIsCancelModalVisible(false);
+  const showCancelModal = () => setIsCancelModalVisible(true);
+
+  const cancelModal = (
+    <EuiConfirmModal
+      data-test-subj="workspaceForm-cancelModal"
+      title={i18n.translate('workspace.form.cancelModal.title', {
+        defaultMessage: 'Discard changes?',
+      })}
+      onCancel={closeCancelModal}
+      onConfirm={() => application?.navigateToApp(WORKSPACE_LIST_APP_ID)}
+      cancelButtonText={i18n.translate('workspace.form.cancelButtonText.', {
+        defaultMessage: 'Continue editing',
+      })}
+      confirmButtonText={i18n.translate('workspace.form.confirmButtonText.', {
+        defaultMessage: 'Discard changes',
+      })}
+      buttonColor="danger"
+      defaultFocusedButton="confirm"
+    >
+      {i18n.translate('workspace.form.cancelModal.body', {
+        defaultMessage: 'This will discard all changes. Are you sure?',
+      })}
+    </EuiConfirmModal>
+  );
+
+  // Number of saved changes will be implemented in workspace update page PR
+  const bottomBar = (
+    <div>
+      <EuiSpacer size="xl" />
+      <EuiSpacer size="xl" />
+      <EuiBottomBar>
+        <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="s">
+              {opType === WORKSPACE_OP_TYPE_UPDATE && (
+                <EuiText textAlign="left">
+                  {i18n.translate('workspace.form.bottomBar.unsavedChanges', {
+                    defaultMessage: '1 Unsaved change(s)',
+                  })}
+                </EuiText>
+              )}
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="m">
+              <EuiButtonEmpty
+                color="ghost"
+                onClick={showCancelModal}
```

Comment:
Do we need to check if there are any unsaved changes, then we show the cancel modal?

Reply:
Since I will track unsaved changes in the update workspace PR, can I include this function in the next PR?